tests/openwisp2/sample_users/tests.py

```diff
@@ -0,0 +1,32 @@
+from openwisp_users.tests.test_admin import (
+    TestBasicUsersIntegration as BaseTestBasicUsersIntegration,
+)
+from openwisp_users.tests.test_admin import (
+    TestMultitenantAdmin as BaseTestMultitenantAdmin,
+)
+from openwisp_users.tests.test_admin import TestUsersAdmin as BaseTestUsersAdmin
+from openwisp_users.tests.test_models import TestUsers as BaseTestUsers
+
+
+class TestUsersAdmin(BaseTestUsersAdmin):
+    app_label = 'sample_users'
+    is_integration_test = True
```

Comment:
According to the tests in [sample_users/tests.py of openwisp-users repository](https://github.com/openwisp/openwisp-users/blob/79021c4c417e0ee20006a628680b6320a230cd99/tests/openwisp2/sample_users/tests.py#L72-L74), you will need to update this. 
```suggestion
class TestUsersAdmin(BaseTestUsersAdmin):
    app_label = 'sample_users'
    is_integration_test = True
    _additional_user_fields = [
        ('social_security_number', '123-45-6789'),
        ('details', 'Example value for detail used during testing.'),
    ]
```

```python

```